### DISCUSSION TITLE
Tags and manual release through GH actions

### DISCUSSION THREAD

**Discussion by pchaganti** [projen/projen#2306] [Q&A] [answered]
Able to do manual releases from local laptop with no issues. Testing out a release through a GH action. Oddly enough it doesnt seem to see the tags on the current repo. It is probably something obvious I am just not seeing as I have been looking at it for a while.

Anyone have any pointers where I am going wrong?

GH actions has perms for:
```
      id-token: write
      contents: write
      actions: write
      packages: write
```

Relevant log from workflow run:
```
yarn run v1.22.19
$ npx projen releaser
� releaser | rm -fr dist
� releaser » bump | condition: ! git log --oneline -1 | grep -q "chore(release):"
� releaser » bump | /opt/hostedtoolcache/node/18.12.1/x64/bin/node /home/runner/work/test_projen/node_modules/projen/lib/release/bump-version.task.js
� Update /home/runner/work/test_projen/cdk-test/package.json to latest resolved version: 0.0.0
� Tags listed on current commit: 
✖ skip version bump on first release
✔ created /home/runner/work/test_projen/cdk-test/dist/changelog.md
✔ outputting changes to /home/runner/work/test_projen/cdk-test/dist/changelog.md
� releaser » build » compile | jsii --silence-warnings=reserved-word
```

![image](https://user-images.githubusercontent.com/6500/207967750-bacd8013-5dfd-4e90-9a6e-c6180c91896a.png)

**Comment by mrgrain** [comment]
You can check the code here: https://github.com/projen/projen/blob/main/src/release/bump-version.ts#L270

The only filter that stands out is the prefix. Have you changed that?

**Comment by pchaganti** [comment, marked as answer]
Found the issue. Needed to run a **_git fetch -a_** after the checkout in the GH workflow environment, to make sure the tags are pulled into the environment. After that everything worked just fine.

thanks!
